node-placeholder-scaler/scaler/scaler.py

```diff
@@ -92,9 +283,33 @@ def main():
 
         # Generate deployment config based on our config
         for pool_name, pool_config in config["nodePools"].items():
-            replica_count = replica_count_overrides.get(
-                pool_name, pool_config["replicas"]
-            )
+            pool_usable_resources = usable_resources_result.get(pool_name + '-pool', {})
```

Comment:
instead of appending '-pool' on to the config name, you can actually parse this out of the config...  since we're going to open source this for the jupyter community i'd rather not force people to use our naming conventions.  :)

`'hub.jupyter.org/pool-name': '<pool-name>'`